executor/src/main/java/com/redhat/service/smartevents/executor/Executor.java

```diff
@@ -8,5 +10,5 @@ public interface Executor {
 
     ProcessorDTO getProcessor();
 
-    void onEvent(CloudEvent event);
+    void onEvent(CloudEvent event, Map<String, String> traceHeaders);
```

Comment:
I need to pass Kafka's headers to our exit point (`ActionInvoker`'s). The code on `main` puts **all** headers into the Cloud Event `data`; however I only want RHOSE's tracing headers in the header; so now I pass them separately.